src/renderer/components/ft-list-video/ft-list-video.js

```diff
@@ -397,9 +425,9 @@ export default defineComponent({
       }
     },
 
-    parseVideoData: function () {
+    parseVideoData: async function () {
       this.id = this.data.videoId
-      this.title = this.data.title
+      this.title = (await this.getDeArrowDataEntry()?.title) ?? this.data.title
```

Comment:
After local testing I found that I got this wrong
Sorry :S 
```suggestion
      this.title = (await this.getDeArrowDataEntry())?.title ?? this.data.title
```

![image](https://github.com/FreeTubeApp/FreeTube/assets/1018543/56ef378e-e3d9-4266-8940-7bce039c13b0)